CHANGES/7500.bugfix

```diff
@@ -0,0 +1 @@
+Removing some unused services from travis config
```

Comment:
It doesn't really matter but I would mark this as "misc" rather than "bugfix".

Don't feel obligated to change it though.